packages/react/src/components/Modal/Modal.tsx

```diff
@@ -387,20 +390,22 @@ const Modal = React.forwardRef(function Modal(
   }, [open, selectorPrimaryFocus, danger, prefix]);
 
   const modalButton = (
-    <button
-      className={modalCloseButtonClass}
-      type="button"
-      onClick={onRequestClose}
-      title={ariaLabel}
-      aria-label={closeButtonLabel ? closeButtonLabel : 'close'}
-      ref={button}>
-      <Close
-        size={20}
-        aria-hidden="true"
-        tabIndex="-1"
-        className={`${modalCloseButtonClass}__icon`}
-      />
-    </button>
+    <div className={`${prefix}--modal-close-button`}>
+      <IconButton
+        className={modalCloseButtonClass}
```

Comment:
Is the wrapper `div` necessary, or could we just combine these classes onto the one element?

Reply:
Earlier I had tried doing this with `modalCloseButtonClass` but find hard to position the button, I think we need this wrapper in order to position button correctly